src/SDKs/ServiceBus/ServiceBus.Tests/Tests/ScenarioTests.QueuesTests.Detail.CRUD.cs

```diff
@@ -0,0 +1,165 @@
+    using Microsoft.Rest.ClientRuntime.Azure.TestFramework;
+    using TestHelper;
+    using Xunit;
+    using System.Threading;
+    public partial class ScenarioTests 
+    {
+        [Fact]
+        public void QueuesCreateGetUpdateDeleteDetail()
+        {
+            using (MockContext context = MockContext.Start(this.GetType().FullName))
+            {
+                InitializeClients(context);
+
+                var location = this.ResourceManagementClient.GetLocationFromProvider();
+
+                var resourceGroup = "v-ajnavtest";
+                //var resourceGroup = this.ResourceManagementClient.TryGetResourceGroup(location);
+                //if (string.IsNullOrWhiteSpace(resourceGroup))
+                //{
+                //    resourceGroup = TestUtilities.GenerateName(ServiceBusManagementHelper.ResourceGroupPrefix);
+                //    this.ResourceManagementClient.TryRegisterResourceGroup(location, resourceGroup);
+                //}
+
+                // Create Namespace
+                var namespaceName = TestUtilities.GenerateName(ServiceBusManagementHelper.NamespacePrefix);
+
+                var createNamespaceResponse = this.ServiceBusManagementClient.Namespaces.CreateOrUpdate(resourceGroup, namespaceName,
+                    new SBNamespace()
+                    {
+                        Location = location,
+                        Sku = new SBSku
+                        {
+                            Name = SkuName.Standard,
+                            Tier = SkuTier.Standard
+                        }
+                    });
+
+                Assert.NotNull(createNamespaceResponse);
+                Assert.Equal(createNamespaceResponse.Name, namespaceName);
+
+                TestUtilities.Wait(TimeSpan.FromSeconds(5));
+
+                // Create Invalid QueueName
+                var queueNameGeneration = TestUtilities.GenerateName(ServiceBusManagementHelper.QueuesPrefix);
+                var queueInvalidName = queueNameGeneration + "$";
+
+                //Invalid QueueName
+                try
+                {
+                    var createinvalidQueueResponse = this.ServiceBusManagementClient.Queues.CreateOrUpdate(resourceGroup, namespaceName, queueInvalidName,
+                                    new SBQueue() { EnableExpress = true, EnableBatchedOperations = true });
+                }
+                catch (Exception e)
+                {
+                    Assert.Equal("Operation returned an invalid status code 'BadRequest'", e.Message);
+                }
+
+                // Create Queue
+                var queueName = TestUtilities.GenerateName(ServiceBusManagementHelper.QueuesPrefix);
+                var createQueueResponse = this.ServiceBusManagementClient.Queues.CreateOrUpdate(resourceGroup, namespaceName, queueName,
+                new SBQueue() { EnableExpress = true, EnableBatchedOperations = true});
+
+                Assert.NotNull(createQueueResponse);
```

Comment:
should verify all properties which are been set during creation.

Reply:
Done!